### DISCUSSION TITLE
Please can anyone help me run this project. Any help will be much appreciated

### DISCUSSION THREAD

**Discussion by LawrenceDarko** [TareqMonwer/Django-School-Management#71] [General]
After inputing the below information in my .env file in the config folder it keeps giving me the below error message

```
 raise ImproperlyConfigured(error_msg)
django.core.exceptions.ImproperlyConfigured: Set the CELERY_BROKER_URL environment variable
```

```
DEBUG=True
SECRET_KEY=12345
EMAIL_USE_TLS=True
EMAIL_HOST=smtp.gmail.com
EMAIL_PORT=587
EMAIL_HOST_USER=darkolawrence@gmail.com
EMAIL_HOST_PASSWORD=*********
DEFAULT_FROM_EMAIL=noreply@gmail.com
BRAINTREE_MERCHANT_ID=5pbrjk4gmzt5m8k6d
BRAINTREE_PUBLIC_KEY=by84t6rfx9nz3v6ks
BRAINTREE_PRIVATE_KEY=202056899b37713b1fa093207160ff2ew
BROKER_URL=amqp://
CELERY_RESULT_BACKEND=db+sqlite:///results.sqlite
```

**Comment by TareqMonwer** [comment]
Hi! @LawrenceDarko .
You should change `BROKER_URL=amqp://` to `CELERY_BROKER_URL=redis://localhost:6379` at second last line.
Let me know this makes your project running.

**Reply by TareqMonwer** [reply]
Thank you for finding the bug, I'll update `.env.example` right away.

**Reply by LawrenceDarko** [reply]
Yeah! Thank you it worked...but it only works when you put the CELERY_BROKER_URL in the settings file

**Reply by TareqMonwer** [reply]
It's because, in `settings.py` file, we are saying that inside the `.env` file, a variable named `CELERY_BROKER_URL` should contain expected URL.

`config/settings.py`
![image](https://user-images.githubusercontent.com/33314452/102506006-deb6b300-40ac-11eb-9278-3a74fe3575da.png)

**Reply by TareqMonwer** [reply]
> Yeah! Thank you it worked...but it only works when you put the CELERY_BROKER_URL in the settings file

I'm glad to help you :)
